test/recipes/70-test_dtls_mtu.t

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env perl
```

Comment:
FYI, I feel unsure this is a test of the PACKET layer.  See `test/README` for the numeric groups.